lib/core/about/about-application-modules/about-application-modules.component.ts

```diff
@@ -43,7 +42,7 @@ export class AboutApplicationModulesComponent implements OnInit {
     @Input() dependencies: any;
 
     constructor(appExtensions: AppExtensionService) {
-        this.extensions$ = appExtensions.references$;
+        appExtensions.references$.subscribe((extensions) => this.extensions = extensions);
```

Comment:
you need to unsubscribe in the ngOnDestroy to avoid memory leak. also, I suggest moving this code to the "ngOnInit" method